data/json/items/tools.json

```diff
@@ -3414,7 +3414,15 @@
     "ammo": "NULL",
     "revert_to": "null",
     "category": "weapons",
-    "use_action": "MOLOTOV"
+    "use_action": {
+        "type": "transform",
+        "target": "molotov_lit",
+        "target_charges": 1,
+        "active": true,
+        "need_fire": 1,
+        "need_fire_msg": "You need a source of fire!",
+        "msg": "You light the Molotov cocktail."
+    }
```

Comment:
Was actually not intended. Suppose I shouldn't be making PR's at my equivalent of 3am on an empty stomach.